### ISSUE TITLE
Several questions for the methods

### ISSUE THREAD

**Issue by yingqihao2022** [DBC-Lab/Brain_MRI_Enhancement#7]
Dear Yue,

I'm Yingqi and I have asked several questions regarding your amazing and wonderful brain enhancement model work before. Thank you so much again for your assistance for the code-related issues.

After reading your passage very carefully, since I'm a beginner in this field, I am confused about some questions as follows:

1. In your methods, you mention that the training foetal dataset "were artefact-free by careful visual inspection". You then added stimulated artefacts to these artefact-free brains. Nonetheless, as Figure 5.b shows, outputs after BME-X is extremely clean miraculously, especially in the gray matter. I am wondering whether the artefact-free training datasets truly did not contain any noise or artefacts inherent in the tissue. If not, what mechanism enables BME-X to produce results that are completely free of noise, particularly in the gray matter?

2. As Figure 8 shows, the BME-X model have the capacity to retain the pathological regions (such as tumours), while your training dataset solely use brains without any pathology, which is similar to unsupervised learning methods. I am wondering if there an explainable reason for this, or you just find this result coincidentally?

I am looking forward to your reply.

Yingqi

**Comment by YueSun814** [comment]
Hi Yingqi,

Good questions! 

1. For training images, we carefully select those with minimal artifacts and noise. However, it is true that the training images cannot be completely artifact-free. Since the majority of training data is clean and artifact-free, and BME-X was trained on a dataset with artificially introduced artifacts in a controlled manner, the network has learned to suppress artifacts while effectively preserving tissue structure. 

As we stated in the paper: "Our approach is rooted in the fundamental assumption that increasing image quality yields clearer and sharper image appearances, accompanied by a convergence of intensity ranges within each tissue type to a single value. Therefore, the complicated task of image reconstruction can be simplified to a tissue classification task. Then, high-quality images can be straightforwardly estimated with the guidance of classification results." This assumption allows BME-X to leverage tissue classification as an intermediate step, ensuring anatomically consistent reconstructions while minimizing the presence of artifacts. By learning the underlying intensity distributions of different tissue types, the model inherently promotes smoother and more homogeneous regions.

2. This occurred because lesions and gliomas were classified as one or a mix of tissue categories (WM, GM, and CSF). It would be better to provide an additional label for abnormal regions in the classification module.

Sorry for the delayed response. I hope this helps!

Yue
